### PR TITLE
clarify

### PR DESCRIPTION
**Pull request by kkraune**
@vekterli please merge

### CODE REVIEW

File: documentation/reference/services-content.html [hunk 1]
```diff
@@ -284,7 +284,9 @@ <h2 id="document-processing">document-processing</h2>
 <h2 id="redundancy">redundancy</h2>
 <p>
 Contained in <code><a href="#content">content</a></code>.
-Defines the total number of copies of each piece of data the cluster will maintain to avoid data loss.
+Defines the total number of copies of each piece of data the cluster will maintain to avoid data loss
+- within a <a href="#group">group</a>.
+If <em>group</em> is not configured, this equals one group, hence all nodes.
```
Comment (vekterli) [inline documentation/reference/services-content.html]: Doesn't group-aware auto scaling of redundancy only apply iff you're running in a hosted setting?

File: documentation/reference/services-content.html [hunk 2]
```diff
@@ -524,6 +526,8 @@ <h2 id="searchable-copies">searchable-copies</h2>
 The benefit is using less memory, trading off availability during transitions.
 Refer to <a href="../proton.html#bucket-move">bucket move</a>.
 </p><p>
+<em>searchable-copies</em> applies within a <a href="#group">group</a>, like <em>redundancy</em>.
+</p><p>
```
Comment (vekterli) [inline documentation/reference/services-content.html]: Same question as above

File: documentation/reference/services-content.html (hunk 1 above)
Comment (kkraune) [reply documentation/reference/services-content.html]: don't know ...

Comment (bratseth) [reply documentation/reference/services-content.html]: Yes, "within a group" is only true on hosted Vespa.

**Pull request closed without merging**